Deploy website + mixer + esp on Kubernetes (#698)
datacommonsorg/website: deployment/gke/README.md

> # Deployt website to multi GKE cluster

Reviewer: ```suggestion
# Deploy Website to Multiple GKE Clusters
```

Author: Done

## Prerequisites

- Create global static IP in GCP.
- Configure the DNS in the domain registra.
- [Grant the required IAM roles](https://cloud.google.com/anthos/multicluster-management/connect/prerequisites#grant_iam_roles).

## One Time Setup

```bash
../generate_yaml.sh <staging | prod>
onetime_setup.sh
```

> This step creates clusters and make all the one time setup, including:

Reviewer: "make all the one time setup" is a bit confusing:

```suggestion
This step creates clusters and runs the one time setup tasks, including:
```

Author: Done

- Enable API in the project.
- Create service account keys.
- Create a cluster in each region
- Setup the config cluster

> Each cluster is a region cluster with 3 nooes in each zone (total 9 nodes).

Reviewer: ```suggestion
Each cluster is a region cluster with 3 nodes in each zone (total 9 nodes).
```

# Deploy process

There are 3 region clusters each with 3 regions (9 nodes), and each node is a
e2-highmem-4 machine type. The region are: `asia-east1`, `europe-west1`, `us-west1`

## Build Docker image and push to Cloud Container Registry

```bash

> cd ../

Reviewer 2: it would be nice to turn these steps into a script once the dust settles

Author: Moved them into scripts (also to ease my testing..)
